### DISCUSSION TITLE
Is it possible to set a default group for a screen?

### DISCUSSION THREAD

**Discussion by luisalcarasr** [qtile/qtile#3386] [Q&A] [answered]
I was wondering, is there an easy way to send a group to a screen from the moment Qtile starts?

**Comment by m-col** [comment, marked as answer]
Yes, for this you can use a startup hook. For example:
```py
from libqtile import qtile, hook

@hook.subscribe.startup
def _():
    # Set initial groups
    if len(qtile.screens) > 1:
        qtile.groups_map["1"].cmd_toscreen(0, toggle=False)
        qtile.groups_map["q"].cmd_toscreen(1, toggle=False)
```
Here I have 2 groups named `"1"` and `"q"` which I am setting to my first and second screens at startup.

**Reply by luisalcarasr** [reply]
Muchas gracias, yesterday I spent all my afternoon trying to figure out how to do it. Thanks again.

**Reply by m-col** [reply]
De nada!
